src/chrome/content/rules/nexusmods.com.xml

```diff
@@ -1,11 +1,11 @@
 <!--
 	Mixed content:
```

Comment:
Is it worth to create a MCB platform rule for them?

Reply:
Maybe but unlikely. I don't think anyone actually has active mixed content blocking disabled after Tor Browser enabled it. I think `platform="mixedcontent"` rulesets are on their way out.